forms.py

```diff
@@ -95,4 +102,17 @@ def __init__(self, *args, **kwargs):
         )
 
     def send_email(self):
-        pass #todo
+        try:
+            params = {
+                'contact_form': self,
+            }
+            html_content = render_to_string("prelogin/_email/contact_form_email.html", params)
+            text_content = render_to_string("prelogin/_email/contact_form_email.txt", params)
+            recipient = settings.CONTACT_EMAIL
+            subject = "Contact Dimagi Request from prelogin"
+            send_HTML_email(subject, recipient, html_content, text_content=text_content,
+                            email_from=settings.DEFAULT_FROM_EMAIL)
+        except Exception:
+            logging.error("Couldn't send pre-login contact email. "
```

Comment:
also, that would add time to the implementation and we want to get a quick iteration of this out the door pronto. this should not hold up a pr.